src/gui/advancedsettings.cpp

```diff
@@ -498,6 +501,9 @@ void AdvancedSettings::loadAdvancedSettings()
     // multiple connections per IP
     m_checkBoxMultiConnectionsPerIp.setChecked(session->multiConnectionsPerIpEnabled());
     addRow(MULTI_CONNECTIONS_PER_IP, tr("Allow multiple connections from the same IP address"), &m_checkBoxMultiConnectionsPerIp);
+    // Disable connection to privileged ports (< 1024)
+    m_checkBoxPrivilegedPortsDisabled.setChecked(session->isConnectToPrivilegedPortsDisabled());
+    addRow(DISABLE_PRIVILEGED_PORTS, (tr("Disable connecting to privileged ports") + ' ' + makeLink("https://libtorrent.org/single-page-ref.html#no_connect_privileged_ports", "(?)")), &m_checkBoxPrivilegedPortsDisabled);
```

Comment:
maybe "Disable connection to peers on privileged port"?

Reply:
I went with `Disable connection to peers on privileged ports`